www/docs/sources_destinations.md

```diff
@@ -19,9 +19,11 @@ Skip then maintains an internal data structure with a timestamp for each token
 that is refreshed at the specified frequency.
 
 When a Skip computation calls `SKStore.getRefreshToken` to get the latest
-timestamp for some refresh token, it incurs a dependency on that token.  Thus,
-it will be re-executed periodically and can produce outputs or intermediate data
-structures which depend on the latest timestamp.
+timestamp for some refresh token, it incurs a dependency on that token.
+Thus, it will be re-executed periodically, enabling you to poll external
+services.  By using the token's timestamp value, you can also reactively
+maintain any outputs or intermediate data structures which depend on the latest
+"current" timestamp.
```

Comment:
Thanks for revising, but I'm still confused, sorry. Reading and thinking about this, I wonder if there is a confusion about the word "timestamp" itself. AFAIK the term "timestamp" means (a record of) the time at which something happened. But here you seem to use it to refer to some other value, not just a time. Or maybe it is just a time value, and I'm not understanding how such a thing gets "used" to produce an output. I mean, the output usually isn't going to actually be a function of the time value, it just needs to get re-evaluated with some frequency, right? Maybe `getRefreshToken` should be named something more like `subscribeToRefreshToken` to indicate that just calling the function in `mapElement` and immediately throwing away the result has the effect of causing later re-evaluations of the `mapElement` and later functions.

Reply:
Right, by "timestamp" here I mean "the integer number of seconds since the unix epoch".  If you're doing something lazy that is a function of the time value, you can use it as is.  I've got a commit in my current working branch which also exposes refresh tokens as a singleton collection, i.e.
`getRefreshTokenEager: (key: string) => EagerCollection<string, RefreshToken>` (along with the current `getRefreshToken : (key:string) => RefreshToken`)
which makes things simpler, I think, for cases where you really do want to set up some eager computation that is a function of the actual time.  (motivated by the oncall rotation example which I've picked back up).

Let's chat more over VC later.  As I see it there are two core use cases for refresh tokens: creating dependencies (without using the actual value) to schedule polls/recomputations, and setting up reactive computations that care about the "current" time. We've focused on the former to this point, but it's only minor additions needed to cover the latter.